R/plot_alignment_table.R

```diff
@@ -28,7 +28,7 @@ plot_alignment_table <- function(data) {
     size_lim <- c(min(data$perc_aum, na.rm = TRUE), max(data$perc_aum, na.rm = TRUE))
     size_range <- c(1, 5)
 
-    if (nrow(data %>% filter(.data$sector == "power")) > 0) {
+    if (nrow(data %>% filter(.data$sector == "power", entity == "portfolio")) > 0) {
```

Comment:
nit: I prefer not to use the pipe when it's not necessary, and use `.data$` to avoid notes
```suggestion
    if (nrow(filter(data, .data$sector == "power", .data$entity == "portfolio")) > 0) {
```